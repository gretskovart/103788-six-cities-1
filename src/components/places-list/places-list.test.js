import React from 'react';
import PlacesList from './places-list.jsx';
import renderer from 'react-test-renderer';

it(`<PlacesList /> renders correctly`, () => {
  const tree = renderer
    .create(<PlacesList offers={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
