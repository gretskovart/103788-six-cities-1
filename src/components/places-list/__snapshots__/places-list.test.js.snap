// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlacesList /> renders correctly 1`] = `null`;
